server/graphql/v1/CollectiveInterface.js

```diff
@@ -1120,6 +1132,35 @@ const CollectiveFields = () => {
         }).then(memberships => memberships.memberCollective);
       },
     },
+    notifications: {
+      type: new GraphQLList(NotificationType),
+      args: {
+        limit: { type: GraphQLInt },
+        offset: { type: GraphQLInt },
+        channel: { type: GraphQLString },
+        type: { type: GraphQLString },
+        active: { type: GraphQLBoolean },
+      },
+      resolve(collective, args) {
+        let where = { CollectiveId: collective.id };
```

Comment:
This is not re-assigned, we can go with `const`

Reply:
Addressed.